connector/apmconnector/transaction.go

```diff
@@ -92,13 +93,28 @@ func (transactions *TransactionsMap) ProcessTransactions() {
 	}
 }
 
-func (transactions *TransactionsMap) GetOrCreateTransaction(sdkLanguage string, span ptrace.Span, resourceMetrics *ResourceMetrics) (*Transaction, string) {
+func GetTransactionKey(traceID string, resourceAttributes pcommon.Map) string {
+	keys := []string{"host.name", "service.name", "container.id", "telemetry.sdk.language"}
```

Comment:
shouldn't we add `service.instance.id` in there? And perhaps also `service.namespace`, as per this [definition](https://github.com/open-telemetry/semantic-conventions/blob/main/model/resource/service_experimental.yaml):

```
MUST be unique for each instance of the same `service.namespace,service.name` pair
(in other words `service.namespace,service.name,service.instance.id` triplet MUST be globally unique).
The ID helps to distinguish instances of the same service that exist at the same time
(e.g. instances of a horizontally scaled service).
```

In theory, this triplet should be sufficient and we shouldn't need host.name, etc. But it looks like this definition is still experimental.

Reply:
Done.  I was just looking at what the OTel demo apps are sending, but in theory we should start seeing those other attributes.